### DISCUSSION TITLE
Missing test domains

### DISCUSSION THREAD

**Discussion by ktlenz** [mhm-ufz/mHM#93] [Installation] [answered]
Hello mHM team, I installed mHM on my Windows 11 machine according to the guide without any errors occurring. However, while trying to test the model with the provided test_domain and test_domain_2 I could not locate these domains.

**Comment by pallavhydro** [comment]
Hi @ktlenz 


Thanks for starting the discussion here.

If you successfully installed mHM, then running `./mhm` from the installation folder should already run mhm with the test domains. The runtime screen should look like the attached screenshot (Note the test domains output paths being shown there).

mHM understands the input/output paths for test domains (or the experiment setup) based on the paths we provide in mhm's main namelist `mhm.nml`.


I hope this helps
Pallav

![Screenshot 2024-05-18 at 12 56 48](https://github.com/mhm-ufz/mHM/assets/58865333/5f0ffc41-55dd-4105-a2ad-3e0d8623a413)

**Reply by ktlenz** [reply]
This unfortunately did not work for me since I alsways got the error message that the directory / files could not be found.

**Comment by MuellerSeb** [comment, marked as answer]
If you installed mhm via [conda](https://mhm-ufz.org/guides/install-win/#installing-the-latest-release) or via [pip](https://pypi.org/project/mhm/) (in a Linux subsystem I assume), you will get an additional terminal command `mhm-download` to solve exactly your problem as described [here](https://mhm-ufz.org/news/2023-06-02-mhm-5-13#python-bindings-and-distribution).

To download and run e.g. test domain 1, you can do this:
```
mhm-download --verbose --domain 1 --path example_domain
mhm example_domain/
```
You can replace `1` with `2` to get test domain 2 and you can specify a target path (other than `example_domain`) as you like.

Hope that helps.
Sebastian

**Reply by ktlenz** [reply]
This worked for me and the model ran without producing any error messages. Thank you.
